test/functional/p2p_blocksonly.py

```diff
@@ -56,33 +56,35 @@ def run_test(self):
             self.nodes[0].p2p.wait_for_tx(txid)
             assert_equal(self.nodes[0].getmempoolinfo()['size'], 1)
 
-        self.log.info('Check that txs from forcerelay peers are not rejected and relayed to others')
-        self.log.info("Restarting node 0 with forcerelay permission and blocksonly")
-        self.restart_node(0, ["-persistmempool=0", "-whitelist=127.0.0.1", "-whitelistforcerelay", "-blocksonly"])
+        self.log.info('Check that txs from relay peers are not rejected and relayed to others')
```

Comment:
"Forcerelay' clearly meant a peer with FORCERELAY permission.
Now that you call it "relay", it's not clear it's about permissions, because we commonly use "relay" in many contexts. I would prefer "RELAY-whitelisted" or something like that hereinafter.

Reply:
You're right, that was confusing. Fixed in https://github.com/bitcoin/bitcoin/pull/19963/commits/3abe9f866c17fb4fde623d410d59f269b673645f